ossm/merge_upstream.sh

```diff
@@ -67,8 +67,15 @@ main () {
     rm -rf bundle/**/*.yaml resources bundle.Dockerfile
     updateVersionsInOssmValuesYaml
     make gen
-    git add bundle resources chart bundle.Dockerfile "$HELM_VALUES_FILE"
+    git add bundle resources chart bundle.Dockerfile vendor "$HELM_VALUES_FILE"
     git -c "user.name=$GIT_USERNAME" -c "user.email=$GIT_EMAIL" commit --no-edit
+  else
```

Comment:
Why do we need this `else`?

Reply:
In case there is no merge conflict, we still want to sync the vendor dir after merging upstream changes.

Reply:
can't we run `make gen` regardless?

Reply:
`make gen` takes a lot of time. I'm not sure if it makes sense to run it when not necessary. if/else makes sure we run the commands only when necessary.